eos/utils/gsl-hacks.cc

```diff
@@ -68,7 +68,7 @@ namespace eos
 
         void gsl_error_handler_hack()
         {
-            gsl_error_handler_t * previous_gsl_error_handler = gsl_set_error_handler(&gsl_error_handler);
+            gsl_error_handler_t * previous_gsl_error_handler __attribute__ ((unused)) = gsl_set_error_handler(&gsl_error_handler);
```

Comment:
If it is unused why do we define it?

Reply:
Otherwise we'll get a warning about a discarded return value!